core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -414,4 +454,11 @@ private long getArtificialUniques(Node node, long artificialUniques) {
         }
         return artificialUniques;
     }
+
+    private long getArtificialUniqueRels(Relationship rel, long artificialUniques) {
+        if (!CypherFormatterUtils.isUniqueRelationship(rel)) {
+            artificialUniques++;
+        }
```

Comment:
If it's not unique we increase number of uniques?

Reply:
Yes this is what I was confused about in our Slack channel here: https://neo4j.slack.com/archives/C032XMUBAHG/p1716906509842369. The Cypher query we build up will add the `UNIQUE IMPORT ID REL` to all relationships which are not already unique, and we want to calculate how many that will be (i.e. how many we make unique artificially), thus the inversion. This info is then used to know how many batches of deletion we need. Does that make sense? I can add a code comment trying to explain this as we both got confused.

Reply:
Still a bit confusing, but if you feel confident I'm happy :).